modules/reloadBidAdapter.js

```diff
@@ -0,0 +1,513 @@
+import {
+  BANNER
+}
+from 'src/mediaTypes';
+import {
+  registerBidder
+}
+from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'reload';
+
+const VERSION_ADAPTER = "1.0";
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: [],
+  supportedMediaTypes: [BANNER],
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params && bid.params.plcmID && bid.params.partID && 'opdomID' in bid.params &&
+      'bsrvID' in bid.params && bid.params.bsrvID >= 0 && bid.params.bsrvID <= 99);
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+
+    var vRequests = [];
```

Comment:
Use let/const where applicable. We recommend using ES6. Please update at other places as well

Reply:
Fixed